nautobot/extras/tests/test_views.py

```diff
@@ -284,7 +284,7 @@ def setUpTestData(cls):
         cls.form_data = {
             "name": "Schema X",
             "slug": "schema-x",
-            "data_schema": '{"type": "object", "properties": {"baz": {"type": "string"}}}',
+            "data_schema": '{"type": "object","properties": {"baz": {"type": "string"}}}',
```

Comment:
Please revert this line.

Reply:
Please keep the line and add the comment to explain for posterity. Thank you!

Reply:
Looks like @bryanculver got there first and added the comment about formatting, no action required on my part. Thanks!

Reply:
😎